specification/azurefleet/AzureFleet.Management/examples/2024-11-01/Fleets_CreateOrUpdate_MinimumSet.json

```diff
@@ -0,0 +1,300 @@
+{
+  "title": "Fleets_CreateOrUpdate_MinimumSet",
+  "operationId": "Fleets_CreateOrUpdate",
+  "parameters": {
+    "api-version": "2024-11-01",
+    "subscriptionId": "1DC2F28C-A625-4B0E-9748-9885A3C9E9EB",
```

Comment:
Should we include real sub id in examples?

Reply:
No, it's changed, but following the subscription format.